### PR TITLE
Made the new frontpage

### PR DESCRIPTION
**Pull request by pholm**

### CODE REVIEW

File: frontend/app/src/components/ChallengeGrid/ChallengeItem.js [hunk 1]
```diff
@@ -60,10 +60,10 @@ const ChallengeItem = ({ item }) => {
           }}
         >
           <div className="ChallengeItem--text">
-            <img class="ChallengeIco open" src={icon.url} />
+            <img className="ChallengeIco open" src={icon.url} />
```
Comment (Earlo) [inline frontend/app/src/components/ChallengeGrid/ChallengeItem.js:63]: `className` is keyword for React components to define class. Normal html components like img, span and so on should use `class`

File: frontend/app/src/pages/Connected/EventInfo/index.js [hunk 2]
```diff
@@ -85,7 +85,7 @@ const EventInfo = (props) => {
             text="Apply here"
           />
           <Link
-            activeClass="active"
+            activeclassName="active"
```
Comment (Earlo) [inline frontend/app/src/pages/Connected/EventInfo/index.js:88]: 🤔

**Review by Earlo — CHANGES_REQUESTED** [review]

File: frontend/app/src/components/ChallengeGrid/ChallengeItem.js (hunk 1 above)
Comment (khenriksson) [reply frontend/app/src/components/ChallengeGrid/ChallengeItem.js:63]: This is actually a great clarification Visa! I've always wondered what's the difference and when to use which.

Comment (pholm) [reply frontend/app/src/components/ChallengeGrid/ChallengeItem.js:63]: Ah, true. I used search-replace in a bit too aggressive way. I think Kasper already fixed this.

File: frontend/app/src/pages/Connected/EventInfo/index.js (hunk 2 above)
Comment (pholm) [reply frontend/app/src/pages/Connected/EventInfo/index.js:88]: Same here.

File: frontend/app/src/components/ChallengeGrid/ChallengeItem.js (hunk 1 above)
Comment (pholm) [reply frontend/app/src/components/ChallengeGrid/ChallengeItem.js]: @Earlo I looked a bit into this. Shouldn't one use className when writing JSX? Also for html elements like div, img etc.

**Comment by pholm** [comment]
Don't merge yet, I just noticed a few mistakes.

**Comment by pholm** [comment]
Can be merged & deployed @Earlo

**Pull request merged**